nls/es/strings.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2013 Adobe Systems Incorporated. All rights reserved.
```

Comment:
yeah, if you can clean it up that would be awesome. It would have been more appropriate if the license was more restrictive and we were actually introducing more significant chunks of logic.
But if you feel we should include that copyright, that's fine by me.